src/test/kotlin/uk/gov/dluhc/printapi/config/LocalStackContainerConfiguration.kt

```diff
@@ -45,12 +45,15 @@ class LocalStackContainerConfiguration {
                     DockerImageName.parse("localstack/localstack:1.2.0")
                 ).withEnv(
                     mapOf(
-                        "SERVICES" to "sqs, dynamodb",
+                        "SERVICES" to "sqs",
                         "AWS_DEFAULT_REGION" to DEFAULT_REGION,
                     )
                 )
                     .withReuse(true)
                     .withExposedPorts(DEFAULT_PORT)
+                    .withCreateContainerCmdModifier {
```

Comment:
Not strictly related to this PR but a failing set of tests on GitHub made me come and look at this ....
We no longer use dynamo in print-api, so I have removed that from the local stack config

Plus, something thats been nagging at me for a while .... we can name our TestContainer containers, which means they dont get random names in Docker Desktop. Whilst we re-use the containers, they are not shared amongst our apps (as you would expect). If you work on several app repos (which we all do), you are likely to have several mysql containers, and several localstack containers. Naming them makes a good deal of sense and we should do this in the other apps too 👍 

![Screenshot 2023-02-02 at 14 52 09](https://user-images.githubusercontent.com/78348259/216357759-b80324d4-9db5-4eb8-b86b-cd2ef07bba89.png)

Reply:
@NathanRussellValtech , minor. Maybe we can add test to the container name, since this container won't be used when we run the application locally. Might prevent some misunderstanding for new-comers.